lib/Header/Header.less

```diff
@@ -162,77 +195,58 @@
 	float: right;
 }
 
-.enyo-locale-non-latin {
-	// Languages with combining accent characters
-	&.enyo-locale-th, // Thai - Test Chars: ฟิ้ไััุุ
-	&.enyo-locale-ar, // Arabic
-	&.enyo-locale-fa, // Farsi
-	&.enyo-locale-ur, // Urdu
-	&.enyo-locale-ku, // Kurdish
-	&.enyo-locale-he, // Hebrew
-	&.enyo-locale-hi, // Hindi
-	&.enyo-locale-ta, // Tamil
-	&.enyo-locale-te, // Telugu
-	&.enyo-locale-kn, // Kannada
-	&.enyo-locale-ml, // Malayalam
-	&.enyo-locale-mr, // Marathi
-	&.enyo-locale-bn, // Bengali
-	&.enyo-locale-pa {// Panjabi
-		.moon-header .moon-header-title {
-			height: 159px;
+//InputHeader classes
+.moon-input-header {
+	.moon-input-header-input-decorator {
+		margin: -1px 0px 0px;
+		padding: 0px;
+		border: 0px;
+		width: 100%;
+		box-sizing: border-box;
+		background-color: transparent;
+
+		& > .moon-input {
+			.moon-header-text;
+			margin: 0px;
+			padding-left: 1px;
+			padding-right: 1px;
+			display: inline-block;
+			box-sizing: border-box;
+			line-height: 1em;
+			color: @moon-input-header-text-color;
+			width: 100%;
+			text-overflow: ellipsis;
+		}
+
+		&.spotlight {
+			& .moon-input {
+				.input-placeholder({
+					color: @moon-spotlight-text-color;
```

Comment:
Can we simplify these rules and specify the color at the lowest level like `.spotlight`, then use `inherit` if the rules don`t get picked up when they should? Seems like a really complicated set of cases that could be restructured to make it easier to follow the cases.